addons/delivery/models/delivery_carrier.py

```diff
@@ -213,6 +213,11 @@ def get_return_label(self,pickings, tracking_number=None, origin_date=None):
         if self.can_generate_return:
             return getattr(self, '%s_get_return_label' % self.delivery_type)(pickings, tracking_number, origin_date)
 
+    def get_label_prefix(self):
+        if hasattr(self, '_%s_get_label_prefix' % self.delivery_type):
+            return getattr(self, '_%s_get_label_prefix' % self.delivery_type)()
```

Comment:
@mart-e we call method if available in delivery integration module else pass default value.
so can you approve this security check?